labelbox/data/annotation_types/ner/document_entity.py

```diff
@@ -0,0 +1,20 @@
+from typing import List
+
+from pydantic import BaseModel, validator
+
+
+class DocumentTextSelection(BaseModel):
+    tokenIds: List[str]
+    groupId: str
+    page: int
+
+    @validator("page")
+    def validate_page(cls, v):
+        if v < 1:
+            raise ValueError("Page must be greater than 1")
```

Comment:
Do we need this validation? If you try to import text selection that doesn't belong to a page, it will be rejected by the import pipeline anyway. I think that one of the stretch goals was to support 0-indexed documents as well (but I don't know where that conversation ended up).



This seems to be duplicated (and incomplete as import pipeline checks for way more things) logic.

Reply:
our sdk documentation here https://docs.labelbox.com/reference/import-document-annotations#entity state page is one-indexed and prediction service does not check for this (it does checks like page counts etc)... @kkim-labelbox any thoughts?